concourse/steps/release.mako

```diff
@@ -497,8 +520,37 @@ if release_notes_md:
       ),
     ),
   )
-  logger.info('added release-notes to component-descriptor:')
+  logger.info('added legacy release-notes to component-descriptor:')
   logger.info(f'{component.resources[-1]}')
+% endif
+
+% if release_notes_docs or subcomponent_release_notes_docs:
+  all_docs = release_notes_docs + subcomponent_release_notes_docs
+  tar_stream = b''.join(release_notes.tarutil.release_notes_docs_into_tarstream(all_docs))
+  tar_digest = f'sha256:{hashlib.sha256(tar_stream).hexdigest()}'
+
+
+  oci_client.put_blob(
+    image_reference=tgt_ref,
+    digest=tar_digest,
+    octets_count=len(tar_stream),
+    data=tar_stream,
+  )
+
+  component.resources.append(
+    ocm.Resource(
+      name='release-notes-archive',
+      version=component.version,
+      type=ocm.ArtefactType.BLOB,
+      access=ocm.LocalBlobAccess(
+        localReference=tar_digest,
+        size=len(tar_stream),
+        mediaType='application/tar',
```

Comment:
should we consider to compress (e.g. gzip) release-notes? I think release-notes will be fairly small; otoh it is a quite common practise 🤔